README.md

```diff
@@ -24,17 +24,17 @@ This module provides Cloud Trace support for Node.js applications. [Google Cloud
 
         npm install --save @google/cloud-trace
 
-2. Include and start the library at the *top of the main script of your application*. It's important that Cloud Trace is the first thing executed so that it can accurately gather data:
+2. If you are running locally, or not on the Google Cloud Platform, set the GCLOUD_PROJECT environment variable. Your project ID is visible in the [Google Cloud Console Console](https://console.cloud.google.com/project?_ga=1.258049870.576536942.1443543237), it may be something like `particular-future-12345`. If your application is [running on Google Cloud Platform](running-on-google-cloud-platform), you don't need to specify the project ID.
```

Comment:
I think the quick start section should precisely describe what is required to run the trace agent the first time it is installed (which will likely not be on GCP). Looping in @justinbeckwith for insight on what would be most helpful here.